readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -610,10 +579,8 @@ def _build_latexmk(self, cwd):
             '-interaction=nonstopmode',
         ]
 
-        cmd_ret = self.build_env.run_command_class(
-            cls=latex_class,
-            cmd=cmd,
-            warn_only=True,
+        cmd_ret = self.run(
```

Comment:
My only worry here is builds that were working before and break now. Is there a way to keep our existing `PDF Written` check, but have that modify the build failure code to explain what changed?

Reply:
I'm not sure to understand your proposal. You want to keep checking for the PDF written in disk and, if not found, fail the build with a custom message instead of the generic saying "You build failed due to a command that failed"?

Reply:
OK. It wasn't hard to implement 😄 . I really like our current pattern with the Celery handler 💯

Reply:
Thanks for working through my meaning here! I think it makes the code a lot better.